### ISSUE TITLE
Update the configuration file to use the new PHP config

### ISSUE THREAD

**Issue by carlos-granados** [Behat/docs#184]
The configuration page 
https://docs.behat.org/en/latest/user_guide/configuration.html
Should be updated to use the new PHP config instead of the old yaml config.
We still want to keep some information about the yaml config, so apart from updating this page to use the PHP config we should add a new section that describes the possibility to use yaml, probably with a full configuration example and some text explaining that this option will probably be deprecated in the future (See https://docs.behat.org/en/latest/user_guide/annotations.html as an example of the text that we used for a similar option of using annotations instead of attributes)

**Comment by carlos-granados** [comment]
@loic425 do you think you will be able to work on this? Let us know if you need any help

**Comment by loic425** [comment]
> @loic425 do you think you will be able to work on this? Let us know if you need any help

Yes sorry, a lot of oss work, but I'll help to finish that. I've opened another PR on the docs days ago.

I''ll work on monday before vacation, I'll try to finish the doc.

**Comment by loic425** [comment]
@carlos-granados I've updated this one
https://github.com/Behat/docs/pull/182

**Comment by loic425** [comment]
@carlos-granados @acoulton 
This PR about configuration is ready to review https://github.com/Behat/docs/pull/192
